treeherder/webapp/graphql/schema.py

```diff
@@ -61,6 +61,16 @@ class Meta:
         model = Repository
 
 
+class OptionCollectionGraph(DjangoObjectType):
+    class Meta:
+        model = OptionCollection
+
+
+class OptionGraph(DjangoObjectType):
```

Comment:
I should have explained this in the pull request, probably.  Maybe I'll update the commit to explain it.